### PR TITLE
Feat/hng 102 subscription renewal reminder template

### LINKED ISSUE #102: [FEAT] EMAIL TEMPLATES -> Subscription Renewal Reminder Template
## Description

_Create a Subscription Renewal Reminder email template according to the design using React Email_

## Acceptance Criteria

- The email template must be responsive
- The email template must be dynamic
- The email template must be done according to the design
- The email tempate must be created using React Email

### Requirements	

The following props should be available for the component

1. Title
2. Name
3. Image
4. Renewal Date
5. Renewal Price
6. Renewal Period
7. Review Subscription Link
8. Renew Subscription Link
10. Any other required props

### Tests

- [ ] Ensure template is responsive
- [ ] Ensure template can receive dynamic data according to what will be displayed
- [ ] Ensure template is displayed properly according to design and requirements

###  Tasks

- [ ] Create one dynamic email template using React Email
- [ ] Add props according to the type of text and data it will display (check the design to confirm what will be displayed)
- [ ] Template must be responsive on Bigger screens, tablet screens and mobile screens
- [ ] Create a unit test to ensure template works as expected

### TimeFrame

**The deadline for submission is Saturday 11:59PM**

# LINKS

[`FIGMA LINK`](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=2156-26950&m=dev).
[`REACT EMAIL DOCS`](https://react.email/docs/components/html).

# IMAGES
Subscription Renewal Reminder (3)
Subscription Renewal Reminder (2)
Subscription Renewal Reminder (1)
Subscription Renewal Reminder

### PR DESCRIPTION
**Pull request by callmestiles**
# Description

Closes #102 

# Changes proposed

## What were you told to do?

Create a Subscription Renewal Reminder email template according to the design using React Email

## What did you do?

Added a new folder to the email templates and created a new email template there with react-email. Also added a test for the email template in the test folder.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos

Figma link: https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=11837-18206&m=dev

![screencapture-localhost-3000-preview-subscription-renewal-reminder-Subscription-2024-07-21-19_42_36]
(https://github.com/user-attachments/assets/2dd324a5-5001-4269-85b2-255d295141e6)
image
image
screencapture-localhost-3000-email-2024-07-19-22_50_08
screencapture-localhost-3000-email-2024-07-19-22_47_41

### CODE REVIEW

**Review by oreoluwa212 — CHANGES_REQUESTED** [review]
Follow Camel case convention for your file names, pull from upstream and ensure all tests passed

**Comment by callmestiles** [comment]
> Follow Camel case convention for your file names, pull from upstream and ensure all tests passed

done

File: src/email/templates/subscriptionRenewalReminder/with-image.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import {
```
Comment (abiona01) [inline src/email/templates/subscriptionRenewalReminder/with-image.tsx:1]: File name is not in camel case

**Review by abiona01 — DISMISSED** [review]

File: src/email/templates/subscriptionRenewalReminder/with-image.tsx (hunk 1 above)
Comment (callmestiles) [reply src/email/templates/subscriptionRenewalReminder/with-image.tsx:1]: Fixed

**Review by oreoluwa212 — APPROVED** [review]

**Pull request merged**